2024-12-09 11:13:44,720 INFO    MainThread:3760 [wandb_setup.py:_flush():79] Current SDK version is 0.18.7
2024-12-09 11:13:44,720 INFO    MainThread:3760 [wandb_setup.py:_flush():79] Configure stats pid to 3760
2024-12-09 11:13:44,721 INFO    MainThread:3760 [wandb_setup.py:_flush():79] Loading settings from C:\Users\eelil\.config\wandb\settings
2024-12-09 11:13:44,721 INFO    MainThread:3760 [wandb_setup.py:_flush():79] Loading settings from C:\Users\eelil\OneDrive\Desktop\Capstone\Machine_Learning_Analysis_of_Bank_Fraud\wandb\settings
2024-12-09 11:13:44,721 INFO    MainThread:3760 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-12-09 11:13:44,721 INFO    MainThread:3760 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-12-09 11:13:44,721 INFO    MainThread:3760 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-12-09 11:13:44,721 INFO    MainThread:3760 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-12-09 11:13:44,721 INFO    MainThread:3760 [wandb_init.py:_log_setup():533] Logging user logs to C:\Users\eelil\OneDrive\Desktop\Capstone\Machine_Learning_Analysis_of_Bank_Fraud\wandb\run-20241209_111344-xujerm4s\logs\debug.log
2024-12-09 11:13:44,721 INFO    MainThread:3760 [wandb_init.py:_log_setup():534] Logging internal logs to C:\Users\eelil\OneDrive\Desktop\Capstone\Machine_Learning_Analysis_of_Bank_Fraud\wandb\run-20241209_111344-xujerm4s\logs\debug-internal.log
2024-12-09 11:13:44,721 INFO    MainThread:3760 [wandb_init.py:_jupyter_setup():479] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001AD965978F0>
2024-12-09 11:13:44,721 INFO    MainThread:3760 [wandb_init.py:init():619] calling init triggers
2024-12-09 11:13:44,721 INFO    MainThread:3760 [wandb_init.py:init():626] wandb.init called with sweep_config: {}
config: {}
2024-12-09 11:13:44,721 INFO    MainThread:3760 [wandb_init.py:init():669] starting backend
2024-12-09 11:13:44,721 INFO    MainThread:3760 [wandb_init.py:init():673] sending inform_init request
2024-12-09 11:13:44,723 INFO    MainThread:3760 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=spawn, using: spawn
2024-12-09 11:13:44,723 INFO    MainThread:3760 [wandb_init.py:init():686] backend started and connected
2024-12-09 11:13:44,727 INFO    MainThread:3760 [wandb_run.py:_label_probe_notebook():1339] probe notebook
2024-12-09 11:13:44,727 INFO    MainThread:3760 [wandb_init.py:init():781] updated telemetry
2024-12-09 11:13:44,747 INFO    MainThread:3760 [wandb_init.py:init():814] communicating run to backend with 90.0 second timeout
2024-12-09 11:13:45,277 INFO    MainThread:3760 [wandb_init.py:init():867] starting run threads in backend
2024-12-09 11:13:45,473 INFO    MainThread:3760 [wandb_run.py:_console_start():2456] atexit reg
2024-12-09 11:13:45,473 INFO    MainThread:3760 [wandb_run.py:_redirect():2305] redirect: wrap_raw
2024-12-09 11:13:45,473 INFO    MainThread:3760 [wandb_run.py:_redirect():2370] Wrapping output streams.
2024-12-09 11:13:45,473 INFO    MainThread:3760 [wandb_run.py:_redirect():2395] Redirects installed.
2024-12-09 11:13:45,475 INFO    MainThread:3760 [wandb_init.py:init():911] run started, returning control to user process
2024-12-09 11:14:14,387 INFO    MainThread:3760 [jupyter.py:save_ipynb():387] not saving jupyter notebook
2024-12-09 11:14:14,387 INFO    MainThread:3760 [wandb_init.py:_pause_backend():444] pausing backend
2024-12-09 11:15:24,362 INFO    MainThread:3760 [wandb_init.py:_resume_backend():449] resuming backend
2024-12-09 11:15:24,671 INFO    MainThread:3760 [jupyter.py:save_ipynb():387] not saving jupyter notebook
2024-12-09 11:15:24,671 INFO    MainThread:3760 [wandb_init.py:_pause_backend():444] pausing backend
2024-12-09 11:17:57,070 INFO    MainThread:3760 [wandb_init.py:_resume_backend():449] resuming backend
2024-12-09 11:17:57,176 INFO    MainThread:3760 [jupyter.py:save_ipynb():387] not saving jupyter notebook
2024-12-09 11:17:57,176 INFO    MainThread:3760 [wandb_init.py:_pause_backend():444] pausing backend
2024-12-09 11:18:11,327 INFO    MainThread:3760 [wandb_init.py:_resume_backend():449] resuming backend
2024-12-09 11:18:13,196 INFO    MainThread:3760 [jupyter.py:save_ipynb():387] not saving jupyter notebook
2024-12-09 11:18:13,196 INFO    MainThread:3760 [wandb_init.py:_pause_backend():444] pausing backend
2024-12-09 11:18:23,656 INFO    MainThread:3760 [wandb_init.py:_resume_backend():449] resuming backend
2024-12-09 11:18:23,671 INFO    MainThread:3760 [jupyter.py:save_ipynb():387] not saving jupyter notebook
2024-12-09 11:18:23,671 INFO    MainThread:3760 [wandb_init.py:_pause_backend():444] pausing backend
2024-12-09 11:19:01,960 INFO    MainThread:3760 [wandb_init.py:_resume_backend():449] resuming backend
2024-12-09 11:19:02,196 INFO    MainThread:3760 [jupyter.py:save_ipynb():387] not saving jupyter notebook
2024-12-09 11:19:02,196 INFO    MainThread:3760 [wandb_init.py:_pause_backend():444] pausing backend
2024-12-09 11:27:34,935 INFO    MainThread:3760 [wandb_init.py:_resume_backend():449] resuming backend
2024-12-09 11:27:34,961 INFO    MainThread:3760 [wandb_setup.py:_flush():79] Current SDK version is 0.18.7
2024-12-09 11:27:34,962 INFO    MainThread:3760 [wandb_setup.py:_flush():79] Configure stats pid to 3760
2024-12-09 11:27:34,962 INFO    MainThread:3760 [wandb_setup.py:_flush():79] Loading settings from C:\Users\eelil\.config\wandb\settings
2024-12-09 11:27:34,962 INFO    MainThread:3760 [wandb_setup.py:_flush():79] Loading settings from C:\Users\eelil\OneDrive\Desktop\Capstone\Machine_Learning_Analysis_of_Bank_Fraud\wandb\settings
2024-12-09 11:27:34,967 INFO    MainThread:3760 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-12-09 11:27:34,967 INFO    MainThread:3760 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-12-09 11:27:34,967 INFO    MainThread:3760 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-12-09 11:27:34,967 INFO    MainThread:3760 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-12-09 11:27:34,967 INFO    MainThread:3760 [wandb_init.py:_log_setup():533] Logging user logs to C:\Users\eelil\OneDrive\Desktop\Capstone\Machine_Learning_Analysis_of_Bank_Fraud\wandb\run-20241209_112734-cxoqj3p2\logs\debug.log
2024-12-09 11:27:34,967 INFO    MainThread:3760 [wandb_init.py:_log_setup():534] Logging internal logs to C:\Users\eelil\OneDrive\Desktop\Capstone\Machine_Learning_Analysis_of_Bank_Fraud\wandb\run-20241209_112734-cxoqj3p2\logs\debug-internal.log
2024-12-09 11:27:34,967 INFO    MainThread:3760 [wandb_init.py:init():619] calling init triggers
2024-12-09 11:27:34,967 INFO    MainThread:3760 [wandb_init.py:init():626] wandb.init called with sweep_config: {}
config: {}
2024-12-09 11:27:34,968 INFO    MainThread:3760 [wandb_init.py:init():644] re-initializing run, found existing run on stack: xujerm4s
2024-12-09 11:27:34,969 INFO    MainThread:3760 [wandb_run.py:_finish():2155] finishing run lhan122-student/Machine_Learning_Analysis_of_Bank_Fraud/xujerm4s
2024-12-09 11:27:34,970 INFO    MainThread:3760 [jupyter.py:save_history():459] not saving jupyter history
2024-12-09 11:27:34,970 INFO    MainThread:3760 [jupyter.py:save_ipynb():387] not saving jupyter notebook
2024-12-09 11:27:34,970 INFO    MainThread:3760 [wandb_init.py:_jupyter_teardown():461] cleaning up jupyter logic
2024-12-09 11:27:34,970 INFO    MainThread:3760 [wandb_run.py:_atexit_cleanup():2420] got exitcode: 0
2024-12-09 11:27:34,970 INFO    MainThread:3760 [wandb_run.py:_restore():2402] restore
2024-12-09 11:27:34,970 INFO    MainThread:3760 [wandb_run.py:_restore():2408] restore done
2024-12-09 11:27:37,253 INFO    MainThread:3760 [wandb_run.py:_footer_history_summary_info():3960] rendering history
2024-12-09 11:27:37,253 INFO    MainThread:3760 [wandb_run.py:_footer_history_summary_info():3992] rendering summary
2024-12-09 11:27:37,257 INFO    MainThread:3760 [wandb_run.py:_footer_sync_info():3921] logging synced files
